ansible/files/admin_api_scripts/pg_upgrade_scripts/complete.sh

```diff
@@ -32,6 +32,45 @@ function execute_extension_upgrade_patches {
     fi
 }
 
+function execute_wrappers_patch {
+    # If upgrading to pgsodium-less Vault, Wrappers need to be updated so that
+    # foreign servers use `vault.secrets.id` instead of `vault.secrets.key_id`
+    UPDATE_WRAPPERS_SERVER_OPTIONS_QUERY=$(cat <<EOF
+  DO \$\$
+  DECLARE
+    server_rec RECORD;
+    option_rec RECORD;
+    vault_secrets RECORD;
+  BEGIN
+    IF EXISTS (SELECT FROM pg_available_extension_versions WHERE name = 'wrappers' AND version = '0.4.6')
```

Comment:
Is 0.4.6 going to be the only impacted version, or do we need to account for future versions as well?

Reply:
Need to account for future versions; was gonna say this should already check for that since all future versions should have previous versions in `pg_available_extension_versions` but that's not the case 🤦

Reply:
Updated the PR and re-testing